### ISSUE TITLE
[Feature Request] One thing is preventing me now to switch from Listary

### ISSUE THREAD

**Issue by derei** [oliverschwendener/ueli#518]
Listary has a very neat feature, for which I found no replacement to the date and I tried everything I could find, all the way to AHK scripts: Is about Quick Switch (open current folder in file dialog when switch from a file manager) - this is their description of the functionality.

Basically, what it does, it opens the path in file dialog (for example when saving a file), just by briefly making the current folder explorer window (the one where we want to save) active. I found this feature so useful that I am wondering why isn't a default integration in Windows.
Listary functionality works equally in File dialog and in Folder dialog (the ugly Browse for Folder, which shows only a tree structure)

Perhaps ueli could copy that functionality.

**Comment by Rereyuop** [comment]
Indeed, this feature from Listary is just super useful. Some of apps won't let you drag a file to send, it required user to find stuff from file dialog. In that case, I locate a picture in file explorer at first, then open the file dialog to click the Listary recommended floating bar. My pic is selected right away. 
I appreciate the interface design of Listary a lot, less complex keyboard combination, but one click.

**Comment by Dhyfer1** [comment]
> Listary has a very neat feature, for which I found no replacement to the date and I tried everything I could find, all the way to AHK scripts: Is about Quick Switch (open current folder in file dialog when switch from a file manager) - this is their description of the functionality.
> 
> Basically, what it does, it opens the path in file dialog (for example when saving a file), just by briefly making the current folder explorer window (the one where we want to save) active. I found this feature so useful that I am wondering why isn't a default integration in Windows.
> Listary functionality works equally in File dialog and in Folder dialog (the ugly Browse for Folder, which shows only a tree structure)
> 
> Perhaps ueli could copy that functionality.

I also agree with @derei. Quick Switch is a very useful Listary feature that no other application launcher has, not even Wox. For the Ueli developer to be clear about how Quick Switch works, I recommend that you watch this video. 

https://youtu.be/9T9-OtRVeUw

And here are some other features that Ueli could copy from Listary, such as Find as you type. https://www.listary.com/docs/getting-started

**Comment by derei** [comment]
any perspective about this feature? or nobody feels it's worth the effort?

**Comment by fabeit** [comment]
One of the most handiest features in Listary, this is one of the reasons why I still use Listary even though I would like to use Ueli or something else. Saves tons of time.

**Comment by GovindKrishnan** [comment]
I agree. This needs attention. Will it be implemented soon? @oliverschwendener

**Comment by eikaramba** [comment]
FYI:  you can install direct folders for that https://www.codesector.com/directfolders

**Comment by derei** [comment]
> FYI: you can install direct folders for that https://www.codesector.com/directfolders

Is a workaround, but not quite the same thing... firstly the free version is rather limited, secondly the paid one is rather expensive and third, it requires more user interaction (selecting the destination manually).
The whole point of computers is to automate repetitive tasks, to find intelligent ways to spare the user of unnecessary steps. Listary developer was ingenious in the approach.

**Comment by eikaramba** [comment]
does it require more steps. at least the quick switching of folders is exactly like in listary. i thought that is what you want.
1) have file dialog open
2) foxus explorer window
3) return to file dialog, boom - you have the same folder from explorer open

**Comment by derei** [comment]
Yeah, that's how listary works indeed. But still... There is the price issue. I'd rather donate those money towards ueli  development, than paying for a software whicj brings only one single feature, which is already being included in a cheaper software along with many other features.
Currently i am using listary 5 and i would have moved to 6 if development wouldn't have been stopped indefinitely.

Ueli drew my attention for one single reason: is being actively developed. I don't like that it uses electron, like at all. It's just bad coding, heavy on resources and inefficient. But at least someone is working on it, which is increasing the chances of adding new useful features.

**Comment by fabeit** [comment]
It's also the fact that there are way too many small programs that perform single tasks, each of them needs a license, or have their own little quirks. Is this function really difficult to implement? Listary has had it for years, so I don't think it's anything that complex.

**Comment by oliverschwendener** [comment]
I'm closing this issue as I have no plans to implement this. If anyone wants to implement this, feel free to reopen.
